pkg/quota_management/expiration_date.go

```diff
@@ -0,0 +1,64 @@
+package quota_management
+
+import (
+	"encoding/json"
+	"gopkg.in/yaml.v2"
+	"strings"
+	"time"
+)
+
+const layout = "2006-01-02 -07:00"
+
+var _ json.Marshaler = &ExpirationDate{}
+var _ json.Unmarshaler = &ExpirationDate{}
+
+var _ yaml.Marshaler = &ExpirationDate{}
+
+var _ yaml.Unmarshaler = &ExpirationDate{}
+
+type ExpirationDate time.Time
+
+func (e *ExpirationDate) UnmarshalJSON(b []byte) error {
```

Comment:
Is there some reason why custom marshal and unmarshal methods are used?
Is it because of the custom layout used? the other comment about using a similar layout to a standarized format might be related to this

Reply:
Yes, it is for the custom layout